src/dtls.c

```diff
@@ -232,20 +234,34 @@ int dtls_connect(struct dtls_ctx *ctx, const char *address, int port)
     return -1;
   }
 
-  ctx->ssl_ctx = SSL_CTX_new(DTLS_client_method());
+  ctx->ssl_ctx = SSL_CTX_new(DTLSv1_2_client_method());
```

Comment:
The `DTLSv1_2_client_method` function is flagged deprecated for me, I'm curious about the change from DTLS_client_method?

Reply:
I was mostly trying to make sure it was using the right version. I can back this out.

Reply:
That makes sense; I've been reading the documentation (https://www.openssl.org/docs/man1.1.0/man3/DTLSv1_2_client_method.html), and I think doing something like:
```
  ctx->ssl_ctx = SSL_CTX_new(DTLS_client_method());
  SSL_CTX_set_min_proto_version(ctx->ssl_ctx, DTLS1_2_VERSION);
```
would probably achieve the same thing, but not using the deprecated function

Reply:
Updated. Thanks!

Reply:
excellent, thanks :)